### PR TITLE
typo in push notification

### PR DESCRIPTION
**Pull request by squallstar**
ref https://github.com/Fliplet/fliplet-studio/issues/2867

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -371,7 +371,7 @@ <h4 class="panel-title collapsed" role="button" data-toggle="collapse" data-pare
                   <li <li><a href="https://help.fliplet.com/article/40-setting-up-push-notifications-for-android" target="_blank">How to set-up push notifications for Android</a></li>
```
Comment (tonytlwu) [inline interface.html]: Typo here too?

Comment (squallstar) [reply interface.html]: fixed!

**Comment by tonytlwu** [comment]
@squallstar It's been nearly a year 🍾

**Comment by tonytlwu** [comment]
@squallstar It doesn't look like this PR in the deployment sheet, and thus won't result in the issue being correctly reviewed and closed.

**Comment by squallstar** [comment]
@tonytlwu thanks, good catch. added!

**Pull request merged**